docs/source/usage/index.rst

```diff
@@ -225,6 +225,58 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console
+---------------
+
+Ironic nodes can be accessed through configuring a web or serial interface.
+This documentation was taken from the Ironic Xena release.
+
+In order to access a node using a serial console, the user must configure the node's console interface. 
+For example, if using ipmi the node's ``console_interface`` must be set to ``ipmitool-socat`` and appropriate ``driver_info`` parameters must be set.
+From here the console can be enabled.
+
++-----------------+----------------------------------------------------+
+|                 | **Actions**                                        |
++-----------------+----------------------------------------------------+
+| Enable Console  | ``openstack baremetal node console enable <node>`` |
++---------------------------+------------------------------------------+
+
+Serial console information is available from the Bare Metal service.  Get
+serial console information for a node from the Bare Metal service as follows::
+
++---------------------------+--------------------------------------------------+
+|                           | **Actions**                                      |
++---------------------------+--------------------------------------------------+
+| Show Console Information  | ``openstack baremetal node console show <node>`` |
++---------------------------+--------------------------------------------------+
+
+ +-----------------+----------------------------------------------------------------------+
+ | Property        | Value                                                                |
+ +-----------------+----------------------------------------------------------------------+
+ | console_enabled | True                                                                 |
+ | console_info    | {u'url': u'tcp://<host>:<port>', u'type': u'socat'}                  |
+ +-----------------+----------------------------------------------------------------------+
+
+If ``console_enabled`` is ``false`` or ``console_info`` is ``None`` then
+the serial console is disabled.
+
+Node serial console of the Bare Metal service is compatible with the
+serial console of the Compute service. Hence, serial consoles to
+Bare Metal nodes can be seen and interacted with via the Dashboard service.
+In order to achieve that, you need to follow the documentation Nova gives.
+
+Note, there can only be one ipmi connection at a time. Note that in order to use ipmi to control a node, the console has to be directed to ttyS0.
+This can be configured via the image.
```

Comment:
What image? Does this mean that a serial console can only be used with certain images, or... ?